deliverables/README.md

```diff
@@ -31,38 +36,154 @@ What is the role of Javascript in a web page?
+text-decoration, text-align, text-indent can all be used to style CSS text. 
 
 ---
 ### Section 3: Git and Github  
 What is Git?  
+Git is a distributed version control systems. In these systems, clients don’t just check out the current version of the files and work from them – they mirror the entire version history. 
+
 What is SCM?  
+A source code manager is a software tool used by teams of programmers to manage source code. 
+SCM is a broader term that encompasses all the processes needed to build, package, and deploy software -- this includes VCS.
+
 What is a VCS?  
+Version Control Systems is software that provides versioning functionality.
+
 Why is Git useful for a developer?  
+Git allows a single developer to make edits while tracking their changes.
+
 Why is Git useful for a team of developers?  
+Git allows for collaboration without confusion of using multiple files.
+
 How do you create a new Git repository for a project locally?  
+The following is a sample of how to create and push the code from your local database to a remote repo. 
+<ul> 
+<li>$ git init</li>
+<li>Initialized empty Git repository in "url"/</li>
+<li>$ cd /usr/local/remoteGit</li>
+<li>$ git init --bare</li>
+<li>Initialized empty Git repository in /usr/local/remoteGit</li>
+<li>$ cd "url"</li>
+<li>$ echo "hello" > hello.txt</li>
+<li>$ git add hello.txt</li>
+<li>$ git commit -m "Initial import" # commit to the local repository</li>
+<li>$ git remote add origin /usr/local/remoteGit # configure the path to the remote repository</li>
+<li>$ git push # push the code to the remote repository</li>
+</ul>
+
 How do you create it on Github?  
+<p>
+<ul>
+<li> git init </li>
+<li>git add .</li>
+<li>git commit -m "first commit"</li>
+<li>git remote add origin       copy the url from the github repo</li>
+<li>git remote -v</li>
+<li>git push origin master </li>
+</ul>
+</p>
+
 How do you commit changes?  
+Commit following the "first commit" and down from the above question
+
 What is the difference between staging and committing changes?  
+Committed means that the data is safely stored in your local database. Staged means that you have marked a modified file in its current version to go into your next commit snapshot. 
+
 What is the difference between committing your changes and pushing them to Github?  
+Committed is on your local database, while pushing them to GitHub stores them on GitHub. 
+
 What is the command to check the status of your current repo in git?  
+You run git status to check a current repo. 
+
 What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
+<ul>
+<li>git log</li>
+<li>--follow</li>
+<li>--source </li>
+<li>--use mail-map</li>
+</ul>
+
 How can you look through your historical commits on the Github website?  
+Use the history tab at the top of the page.
```

Comment:
Click the link to "[N] commits" just above the file list at the repo's root url.